### ISSUE TITLE
Add default placeholder

### ISSUE THREAD

**Issue by miina** [xwp/wp-customize-object-selector#15]
Currently when placeholder is missing, it breaks single select2 field when trying to remove an already selected object.

**Comment by westonruter** [comment]
So shouldn't it be _required_?

**Comment by miina** [comment]
Maybe having a default placeholder would be a good option? Then it would always be there but would be optional when configuring a field. So required within the plugin but optional for field configuring.

**Comment by westonruter** [comment]
OK, so then the issue should be, “Add default placeholder”, since placeholders are required in some cases?

**Comment by miina** [comment]
Yes, sorry for the ambiguity, changed. Just in case: is this still actual or was it part of #20?

**Comment by westonruter** [comment]
@miina no, it wasn't part of #20. Still an open issue.
